requirements.txt

```diff
@@ -35,7 +35,7 @@
 #   deactivate
 #
 
-Django>=1.9,<1.9.99
+Django>=1.9,<1.10
```

Comment:
that is interesting, and somewhat weird.  I'll update this patch to align them all to be that way